File: BocetoV2/DaoMock/TareaDaoMock.java
import java.time.LocalDateTime;

public class TareaDaoMock extends TareaDao {
    
    public TareaDaoMock() {
        super();
        
        // Datos inicializados de prueba
        Tarea t1 = new Tarea("Comprar comida", "Ir al supermercado", LocalDateTime.now().plusDays(1), EtiquetaTarea.HOGAR);
        Tarea t2 = new Tarea("Estudiar Java", "Repasar POO", LocalDateTime.now(), EtiquetaTarea.ESTUDIO);
        
        this.guardar(t1);
        this.guardar(t2);
    }
}

File: App/AppV3/app/src/main/java/com/pmdm/planify/data/daomocks/TareaDaoMock.kt
package com.pmdm.planify.data.daomocks

import com.pmdm.planify.data.mocks.TareaMock
import com.pmdm.planify.models.EtiquetaTarea
import java.time.LocalDateTime

class TareaDaoMock {
    // Simulamos la tabla de Tareas (una lista mutable)
    val tareas = mutableListOf(
        TareaMock(
            titulo = "Comprar comida",
            descripcion = "Ir al supermercado",
            fecha = LocalDateTime.now().plusDays(1),
            etiqueta = EtiquetaTarea.HOGAR
        ),
        TareaMock(
            titulo = "Estudiar Java",
            descripcion = "Repasar POO",
            fecha = LocalDateTime.now(),
            etiqueta = EtiquetaTarea.ESTUDIO
        )
    )
}